frb_codegen/src/lib.rs

```diff
@@ -43,44 +43,63 @@ mod transformer;
 pub mod utils;
 use error::*;
 
+/// When the API is only defined in 1 rust file(block), take this one for generation, where `config`
+/// is the instance containing all information to the API file(block), and `all_symbols` contains
+/// all unique APIs defined in the file mentioned above.
+/// If APIs are defined in more than 1 rust file(block), use `frb_codegen_multi` instead.
 pub fn frb_codegen(config: &config::Opts, all_symbols: &[String]) -> anyhow::Result<()> {
     frb_codegen_multi(&[config.clone()], 0, all_symbols)
 }
 
-/// the `all_configs` here is used only for multi-blocks, because the current block needs information from all other blocks，
-/// and `index` refers to the index of current block to deal with.
+/// This function is used only for cases with multi-blocks when there are
+/// more than 1 API block. And because the current block to deal with needs information
+/// from all other blocks， so `all_configs` is used here,
+/// with `index` refers to the place of the current block to deal with.
+/// For details on how to take advantage of multi-blocks, please refers to this article: https://cjycode.com/flutter_rust_bridge/feature/multiple_files.html
```

Comment:
```suggestion
/// This function is used only for cases with multi-blocks when there are
/// more than one API block. 
/// In addition, because the current block to deal with needs information
/// from all other blocks, `all_configs` is used here,
/// with `index` referring to the place of the current block to deal with.
/// For details on how to take advantage of multi-blocks, please refers to this article: https://cjycode.com/flutter_rust_bridge/feature/multiple_files.html
```